index.js

```diff
@@ -45,63 +53,48 @@ function disconnect() {
  * Connect to a mongodb
  *
  * @param {string} mongoConnectionString complete mongo connection string. e.g. user:password@mongodb:27017/database
- * @param {object} [options] mongo client options see link below
- * https://mongodb.github.io/node-mongodb-native/1.4/driver-articles/mongoclient.html?highlight=server#db-a-hash-of-options-at-the-db-level-overriding-or-adjusting-functionality-not-supported-by-the-url
+ * @param {object} [options] mongo client options; see README
  *
- * @returns {Boolean} If a new connection has been established
+ * @returns {Promise<Mongoose>} Returns the connection
  */
 function connect(mongoConnectionString, options = {}) {
   // Generate options
   options = { ...defaultOptions, ...options }
-  logger.info('**** mongoose-harakiri: Connecting with these options;', options)
+  logger.info('**** mongoose-harakiri: Connecting with these options:')
+  logger.info('**** mongoose-harakiri: ', options)
```

Comment:
you don't need a space after `:`, the new parameter in logger already gives a space.